uncompyle6/semantics/n_actions.py

```diff
@@ -267,7 +267,10 @@ def n_const_list(self, node):
                 if elem == "add_value":
                     elem = elem[0]
                 if elem == "ADD_VALUE":
-                    value = "%r" % elem.pattr
+                    if self.version[0] == 2:
```

Comment:
There is also Python 1.x. The convention used so far is `self.version < (3, 0)` which I think is clearer.